src/Config/Auth.php

```diff
@@ -127,6 +127,17 @@ class Auth extends BaseConfig
         'tokens',
     ];
 
+    /**
+     * --------------------------------------------------------------------
+     * Login validation rules
+     * --------------------------------------------------------------------
+     * Returns the rules that should be used for user login validation.
+     */
+    public array $loginRules = [
+        'email'    => 'required|valid_email',
```

Comment:
According to the RFC, the maximum length of an email address is 254 characters.

```suggestion
        'email'    => 'required|valid_email|max_length[254]',
```